Userland/Applications/Escalator/EscalatorWindow.cpp

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2022, Ashley N. <dev-serenity@ne0ndrag0n.com>
+ * Copyright (c) 2022, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include "EscalatorWindow.h"
+#include <AK/Assertions.h>
+#include <Applications/Escalator/EscalatorGML.h>
+#include <LibCore/File.h>
+#include <LibCore/SecretString.h>
+#include <LibCore/System.h>
+#include <LibGUI/FileIconProvider.h>
+#include <LibGUI/Label.h>
+#include <LibGUI/MessageBox.h>
+#include <LibGUI/Widget.h>
+#include <stdlib.h>
+#include <unistd.h>
```

Comment:
One last nitpick, I don't think these are used anymore thanks to the use of all the fun LibCore helpers :)

Reply:
`environ` is used from unistd.h to carry forward the user's execution environment when calling `exec`. However, stdlib doesn't appear to be used in this file. I have removed stdlib.h as a result.